port-forward-cloud-apis-locally.ps1

```diff
@@ -0,0 +1,60 @@
+﻿#############################################################
+### Port forward cloud env locally                        ###
+### usage:                                                ###
+###  ./port-forward-cloud-apis-locally.ps1 -n <Namespace> ###
+#############################################################
+
+
+param(
+        [Parameter()]
+        [string]$Namespace
+    )
+
+function constructServicesToPortsMap()
+{
+    $Map = @{
+    "bapi" = "5100"
+    "dapi" = "5101"
+    "isapi" = "5102"
+    "oapi" = "5103"
+    "ordapi" = "5104"
+    }
+    return $Map
+}
+
+$ServicesToPorts = constructServicesToPortsMap
+
+
+$JobIds=@()
+try
+{
+    $ServicesToPorts.keys | ForEach-Object {
+
+        $ScriptBlock = { 
+        param
+        (
+            $key,
+            $value,
+            $Namespace
+        )
+        kubectl port-forward service/gpitfutures-bc-$key $($value):$($value) -n $Namespace
+        }
+
+        $Job = Start-Job -ScriptBlock $ScriptBlock -ArgumentList $_, $ServicesToPorts[$_], $Namespace -Name $_
+        $JobIds+=$Job.Id
+    }
+    write-host "Starting..." 
+    Read-Host -Prompt "To stop port forwarding, please either press Enter or kill this process (Ctrl + C)."
+}
+
+catch { }
+
+finally
+{
+    Write-Host "`nStarting cleanup..."
+    foreach ($Job in $JobIds)
+    {
+        Stop-Job -Id $Job

```

Comment:
what does this stop-job refer to? the port forwarding, or the pod/job that we're looking at?

Reply:
to the  job that spun a background task which does the `kubectl port forward...` - this is just stopping it so that its not running even after you exited this script.